diff --git a/ash/system/privacy_hub/geolocation_privacy_switch_controller.cc b/ash/system/privacy_hub/geolocation_privacy_switch_controller.cc
index 1b65a032269a309f52e05165442ff3eb472813d9..36b6814bdc30eafddca78616af063729abe41065 100644
--- a/ash/system/privacy_hub/geolocation_privacy_switch_controller.cc
+++ b/ash/system/privacy_hub/geolocation_privacy_switch_controller.cc
@@ -24,7 +24,8 @@
 namespace ash {
 
 GeolocationPrivacySwitchController::GeolocationPrivacySwitchController()
-    : session_observation_(this) {
+    : session_observation_(this),
+      cached_access_level_(GeolocationAccessLevel::kAllowed) {
   session_observation_.Observe(Shell::Get()->session_controller());
 }
 
